src/sentry/migrations/0216_cdc_setup_replication_index.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 1.11.29 on 2021-06-30 18:51
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = True
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = False
+
+    dependencies = [
+        ("sentry", "0215_fix_state"),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name="group",
+            unique_together={("project", "id"), ("project", "short_id")},
+        ),
```

Comment:
Does this create the index concurrently ?

Reply:
No, but these already exist in Saas. I guess we might want this to be concurrent for ST though.

Reply:
Hhmm. I thought setting atomic=False would help but I guess that does not create the index concurrently. Since this PR has already been merged, is there any followup I should do?

Reply:
It's probably worth modifying the migration in a separate pr, since this hasn't gone out to ST or on-prem yet. Here's an example of how to create this concurrently: https://github.com/getsentry/sentry/blob/3e0dc49ef4c3fc14865b641c9c6ee557e733c3b9/src/sentry/migrations/0213_rule_project_status_owner_index.py

Reply:
Ok. Let me give it a shot. Thanks for the input